client/actions/actions.js

```diff
@@ -44,19 +58,144 @@ const fetchAccounts = (username) => (dispatch) => {
   fetch('/connect/get', options)
     .then(res => res.json())
     .then(res => {
+      dispatch(sortingTransStart())
       dispatch(getMemberData(res))
+      dispatch(sortTransactions(res.transactions))
+      dispatch(changeView('member'))
     })
     .catch(err => console.error(err))
 }
 
+
+// Need to sort all of the transactions in the store by date descending
+const sortTransactions = (unsorted) => dispatch => {
+
```

Comment:
This should be a  _read_ operation instead of a _write_ operation. Check out [`reselect`](https://github.com/reactjs/reselect).